target/linux/bcm53xx/image/Makefile

```diff
@@ -244,6 +244,13 @@ define Device/linksys-ea9500
 endef
 # TARGET_DEVICES += linksys-ea9500
 
+define Device/phicomm-k3
+  DEVICE_TITLE := PHICOMM K3
+  DEVICE_PACKAGES := $(BRCMFMAC_4366C0) $(USB3_PACKAGES) phicommk3-screenctrl phicommk3-firmware
```

Comment:
Probably you should define something like this somewhere:
```
BRCMFMAC_4366C0 := $(IEEE8021X) kmod-brcmfmac phicommk3-firmware
```
Then phicommk3-firmware should have a more generic name.

Reply:
I can do this part.- -